ui/components/component-library/avatar-base/avatar-base.tsx

```diff
@@ -54,7 +59,7 @@ export const AvatarBase = forwardRef(
         borderRadius={BorderRadius.full}
         variant={fallbackTextVariant}
         textTransform={TextTransform.Uppercase}
-        {...{ backgroundColor, borderColor, color, ...props }}
+        {...{ backgroundColor, borderColor, color, ...props }} // TODO: Fix type error to do with spreading ...props in Box types
       >
         {children}
       </Text>
```

Comment:
Screencasts show no visual deprecation of `AvatarBase` and that `Box` props still work as intended

### Before

https://github.com/MetaMask/metamask-extension/assets/8112138/ba8a22e5-0de2-4910-85e9-863dee8d7b0a

### After

https://github.com/MetaMask/metamask-extension/assets/8112138/bea3741e-219e-4a53-9d76-01cd04af4e24